metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

```diff
@@ -276,3 +282,18 @@ object CommandHTMLFormat {
     }
   }
 }
+
+sealed trait AutoImportBuildKind
+object AutoImportBuildKind {
+  object Off extends AutoImportBuildKind
+  object Initial extends AutoImportBuildKind
+  object All extends AutoImportBuildKind
```

Comment:
```suggestion
  case object Off extends AutoImportBuildKind
  case object Initial extends AutoImportBuildKind
  case object All extends AutoImportBuildKind
```
Nit, can help with logging/debugging (nicer to work with as enumeration). :slightly_smiling_face: